msal/src/Microsoft.Identity.Client/CacheV2/Schema/StorageJsonValues.cs

```diff
@@ -0,0 +1,41 @@
+﻿// ------------------------------------------------------------------------------
+// 
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+// 
+// This code is licensed under the MIT License.
+// 
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+// 
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+// 
+// ------------------------------------------------------------------------------
+
+namespace Microsoft.Identity.Client.CacheV2.Schema
+{
+    internal static class StorageJsonValues
+    {
+        public const string AuthorityTypeAdfs = "ADFS";
+        public const string AuthorityTypeMsa = "MSA";
+        public const string AuthorityTypeMsSts = "MSSTS";
+        public const string AuthorityTypeOther = "Other";
+        public const string CredentialTypeRefreshToken = "RefreshToken";
+        public const string CredentialTypeAccessToken = "AccessToken";
+        public const string CredentialTypeIdToken = "IdToken";
+        public const string CredentialTypeOther = "Other";
```

Comment:
what about B2C? Does that need to be included?

Reply:
It will need to be.  This is the first run and msalc++ doesn't support b2c now so it wasn't in there.  as we fill in scenarios and test cases we'll need to expand support here.